front-end/src/app/reports/reports-routing.module.ts

```diff
@@ -67,6 +67,7 @@ const routes: Routes = [
     path: 'f3x/memo/:reportId',
     component: ReportLevelMemoComponent,
     canActivate: [ReportIsEditableGuard],
+    resolve: { report: ReportResolver },
```

Comment:
This can be removed. Because an issue where the ReportIsEditableGuard needed to have the full report object, it pulls the report like the ReportResolver and makes it available in the ngrx store. So in effect, only the ReportIsEditableGuard or the ReportResolver is needed but not both, otherwise we end up making 2 API calls to the backend where only one is needed.